src/pybel/manager/models.py

```diff
@@ -419,8 +416,8 @@ class Network(Base):
     created = Column(DateTime, default=datetime.datetime.utcnow)
     blob = Column(LargeBinary(LONGBLOB), doc='A pickled version of this network')
 
-    nodes = relationship('Node', secondary=network_node, lazy="dynamic")  # backref='networks'
-    edges = relationship('Edge', secondary=network_edge, lazy="dynamic")  # backref='networks'
+    nodes = relationship('Node', secondary=network_node, lazy="dynamic")
+    edges = relationship('Edge', secondary=network_edge, lazy="dynamic")

```

Comment:
why can't we use the keyword argument: `cascade="all, delete-orphan"`?

Reply:
The cascade won't work because we might have multiple parent / child tables